lib/main_publiccloud.pm

```diff
@@ -133,10 +133,10 @@ sub load_latest_publiccloud_tests {
             loadtest('publiccloud/cloud_netconfig', run_args => $args);
         }
         else {
+            loadtest("publiccloud/check_services", run_args => $args) if (get_var('PUBLIC_CLOUD_SMOKETEST'));
```

Comment:
in maintenance updates flow you moved it to img_proof which is ok . in latest you moved it here which would multiply it into several different different jobs. I have two issues here : 
- why it different from maintenance runs ?
- why we need to do it in several tests 
I think we need to do the same in latest  as you did in maintenance

Reply:
Let's talk about it but in a nutshell: Test suite `img_proof` in latests runs single test module, e.g. in one test module the machine is created, used and destroyed. Adding additional test module would break the logic.

Reply:
let's solve this in next PR